src/NServiceBus.AcceptanceTests/Core/Sagas/When_adding_state_to_context.cs

```diff
@@ -39,6 +39,7 @@ public SagaEndpoint()
             {
                 EndpointSetup<DefaultServer>(c =>
                 {
+                    c.UsePersistence<InMemoryPersistence>();
```

Comment:
no idea. but this was not introduced in this PR so should be raised on a diff issue